### DISCUSSION TITLE
Nice feature of Copilot: score of the results "debug.showScores": true

### DISCUSSION THREAD

**Discussion by Anselmoo** [community/community#15784] [Copilot Conversations]
The advanced feature is pretty awesome and easily helps to classify the results!

```json
"github.copilot.advanced": {
    "debug.showScores": true
  }
```

<img width="931" alt="image" src="https://user-images.githubusercontent.com/13209783/165926061-5fd5590f-38f5-471b-ac56-72a1a2bff4e3.png">

**Comment by roccho-dev** [comment]
check

**Comment by fisforfaheem** [comment]
any more features?? please share so i can add in vscode
